package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "chalk": "^2.4.2",
-    "ember-template-recast": "^3.2.8",
+    "ember-template-recast": "^3.2.1",
```

Comment:
This seems slightly odd? the before has ^3.2.8 and this PR downgrades it to 3.2.1, is that right?

Reply:
Oups it’s a typo, I meant ^3.3.1 (as in yarn.lock)